opendebates/admin.py

```diff
@@ -78,20 +110,33 @@ class CandidateAdmin(ModelAdmin):
 
 @register(models.Flag)
 class FlagAdmin(ModelAdmin):
-    list_display = [f.name for f in models.Flag._meta.fields]
+    list_display = ('id', 'prefix', 'to_remove', 'duplicate_of', 'voter', 'reviewed', 'note')
```

Comment:
I would have used a literal tuple for all of them, except that Submission just has way too many fields.